### PR TITLE
Add labels to LogicalCluster

### PR DESCRIPTION
**Pull request by awmackowiak**
Screenshot 2022-11-16 at 14 05 09

### CODE REVIEW

File: vaas/vaas/cluster/admin.py [hunk 1]
```diff
@@ -194,6 +195,16 @@ def get_tags(self, obj):
 
     get_tags.short_description = 'Current vcls'
 
+    def labels(self, obj):
+        labels_list_html = ''
+        if obj.labels:
+            for label in obj.labels:
+                labels_list_html += "<div class='span13 text-center'>%s</div>" % label
+            return format_html(labels_list_html)
+        return
```
Comment (ziollek) [inline vaas/vaas/cluster/admin.py:204]: I pretty sure that single outer scope return would be enough:
```
return format_html(labels_list_html)
```

File: vaas/vaas/cluster/models.py [hunk 2]
```diff
@@ -21,26 +21,44 @@ class LogicalCluster(models.Model):
...
+    def labels(self):
+        if self._labels is None:
+            self._labels = self._get_set(self.labels_list)
+        return self._labels
+
+    @labels.setter
+    def labels(self, labels):
+        self._labels, self.labels_list = self._prepare_set_and_json(labels)
+
+    def _get_set(self, value):
```
Comment (ziollek) [inline vaas/vaas/cluster/models.py:50]: A little ambiguous name, don't you think?

File: vaas/vaas/cluster/models.py [hunk 3]
```diff
@@ -21,26 +21,44 @@ class LogicalCluster(models.Model):
...
-        if isinstance(versions, list):
-            self._current_vcls = set(versions)
-            self.current_vcl_versions = json.dumps(versions)
+        self._current_vcls, self.current_vcl_versions = self._prepare_set_and_json(versions)
+
+    @property
+    def labels(self):
+        if self._labels is None:
+            self._labels = self._get_set(self.labels_list)
+        return self._labels
```
Comment (ziollek) [inline vaas/vaas/cluster/models.py:44]: It's nice that you extract part of the repeated code, but why not the entire?

```
def _get_parsed_field(self, field, values):
    if field is None:
        field = self._get_set(values)
    return field
```

**Review by ziollek — CHANGES_REQUESTED** [review]

File: vaas/vaas/cluster/models.py (hunk 2 above)
Comment (awmackowiak) [reply vaas/vaas/cluster/models.py:50]: Yes. I have had same feeling. I don't know what name will fit here. Do you have some proposition?

Comment (awmackowiak) [reply vaas/vaas/cluster/models.py]: Ok. Never mind. I refactor in different way :)

File: vaas/vaas/cluster/models.py (hunk 3 above)
Comment (awmackowiak) [reply vaas/vaas/cluster/models.py]: True. I have been change it.

**Review by ziollek — APPROVED** [review]

**Pull request merged**